CMakeLists.txt

```diff
@@ -451,5 +451,8 @@ include_directories(.)
 # generates an installable makefile.
 install(FILES velox/type/Type.h DESTINATION "include/velox")
 
+include_directories(SYSTEM
+                    ${CMAKE_CURRENT_SOURCE_DIR}/third_party/simdjson/include)
+
```

Comment:
Please use the target created by the library and link it to the targets that actually need it using `target_link_library`. The use of global scope functions like `include_directories` is deprecated in modern cmake. (I am aware that we still use them but we are working on it ;) )